src/bors/handlers/trybuild.rs

```diff
@@ -137,14 +138,22 @@ pub(super) async fn command_try_build(
                         cancelled_workflow_urls,
                     ),
                 )
-                .await
+                .await?;
+            db.insert_comment(
```

Comment:
I was thinking of attaching the tags directly to the `Comment` struct, and then having a function that would take the GH client and a DB handle and insert both. But since we only need the tags at a single place right now, it would be a bit overkill, so let's leave it like it is for now, and I'll just leave this comment here for future reference.